### PR TITLE
feat(pins-views): add mvp views to common-service

### PR DESCRIPTION
**Pull request by swdpcomputing**
To be uploaded to npm

### CODE REVIEW

File: services/views/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+src/public/javascripts/main.js
```
Comment (shepda) [inline services/views/.eslintignore:2]: It doesn't look like we have this file so perhaps don't need to include it here?

File: services/views/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix",
+    "format": "prettier --write './**/*.js'",
+    "test": "jest --coverage --maxWorkers=4"
+  },
+  "dependencies": {
+    "govuk-frontend": "^3.13.0",
+    "pino": "^6.7.0"
```
Comment (shepda) [inline services/views/package.json:14]: It doesn't look like we're doing any logging so do we need pino as a dependency?

File: services/views/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  "devDependencies": {
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.14.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "lint-staged": "^10.5.0",
+    "patch-package": "^6.2.2",
+    "pino-pretty": "^4.3.0",
```
Comment (shepda) [inline services/views/package.json:25]: It doesn't look like we're doing any logging so do we need pino-pretty as a dependency?

File: services/views/package.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  "devDependencies": {
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.14.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "lint-staged": "^10.5.0",
+    "patch-package": "^6.2.2",
```
Comment (shepda) [inline services/views/package.json:24]: Are we using patch-package?

File: services/views/package.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  "description": "nunjucks views used by pins",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix",
+    "format": "prettier --write './**/*.js'",
+    "test": "jest --coverage --maxWorkers=4"
+  },
+  "dependencies": {
+    "govuk-frontend": "^3.13.0",
```
Comment (shepda) [inline services/views/package.json:13]: It doesn't look like govuk-frontend is being used in this module but is being called from the app consuming the module, so perhaps we don't need to include it here?

File: services/views/.eslintignore (hunk 1 above)
Comment (swdpcomputing) [reply services/views/.eslintignore:2]: Removed

File: services/views/.eslintrc [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base", "prettier"],
+  "plugins": ["jest", "prettier"],
+  "root": true,
+  "ignorePatterns": ["node_modules/**", "src/public/javascript/**"],
```
[the remark this answers is not in the document]
Comment (swdpcomputing) [reply services/views/.eslintrc:9]: Removed

File: services/views/package.json (hunk 2 above)
Comment (swdpcomputing) [reply services/views/package.json:14]: Removed

File: services/views/package.json (hunk 3 above)
Comment (swdpcomputing) [reply services/views/package.json:25]: Removed

File: services/views/package.json (hunk 4 above)
Comment (swdpcomputing) [reply services/views/package.json:24]: No, Removed

File: services/views/package.json (hunk 5 above)
Comment (swdpcomputing) [reply services/views/package.json:13]: I was wondering. Removed.

**Review by shepda — APPROVED** [review]
LGTM pending comment in Slack

**Pull request merged**